### ISSUE TITLE
jsoninclude: Hide select box for single directives

### ISSUE THREAD

**Issue by duncandewhurst** [OpenDataServices/sphinxcontrib-opendataservices#40]
The select box is useful when there are consecutive directives but useless when there is only one directive.

**Comment by jpmckinney** [comment]
I don't think the directive creates the select box, but rather JS code, [like this for OCDS](https://github.com/open-contracting/standard/blob/8a5560a95f7789a19a263ddd78a9d07224ce9635/docs/_static/script.js#L90-L110). Perhaps the project with the select box has incorrect logic.

**Comment by duncandewhurst** [comment]
Yes, I think the [Javascript files](https://sphinxcontrib-opendataservices.readthedocs.io/en/latest/jsoninclude/#javascript) linked in the documentation for the directive need to be updated with the correct logic. At least, the examples on that page show a selection box even when there is only one directive.

Edit: For future reference, I created this issue whilst working on the [OFDS documentation](https://open-fibre-data-standard.readthedocs.io).

**Comment by jpmckinney** [comment]
Aha - I might have simply fixed this for OCDS. Feel free to copy that code into this project (with maybe a link in a comment, so it's easy to check if the source has changed).
